### DISCUSSION TITLE
ModuleNotFoundError

### DISCUSSION THREAD

**Discussion by Morty1902** [TomSchimansky/CustomTkinter#2131] [Q&A]
When running the code it produces errors like the one in the photo below. I will be very grateful if someone helps
![Screenshot 2023-12-07 104719](https://github.com/TomSchimansky/CustomTkinter/assets/88483327/fbbc9b82-3df4-4225-a35e-478086b4e4bb)

**Comment by Akascape** [comment]
Again the same issue reported by every new user.

@Morty1902 Please install the packaging module first by `pip install packaging`.


@TomSchimansky Please update the package!
